jsonrpc-types/src/rpctypes/basic/integer.rs

```diff
@@ -84,11 +88,11 @@ macro_rules! impl_convert_with_uint {
                 Integer::new(data as u64)
             }
         }
+
         impl TryInto<$uint> for Integer {
```

Comment:
You do not have to add a trait.
IMHO, just impl a method is better: `fn try_into_uint<T> -> Result<T, Error>` or `fn try_into_uint<T> ->Option<T>`.
Do not let the old code to restrict u.

Reply:
That trait is now shadowed.